stdlib/ballerina-http/src/main/ballerina/http/auth_handler_registry.bal

```diff
@@ -15,50 +15,72 @@
 // specific language governing permissions and limitations
 // under the License.
 
-@Description {value:"Representation of the Http Auth Handler Registry"}
-@Field {value:"httpAuthHandlers: map for auth handlers. key = auth provider id"}
+documentation {
+    Representation of the Http Auth Handler Registry
```

Comment:
no periods at the end of doc comments in this file. Needs a new line at the end too :)